man/foreman-debug.8.asciidoc

```diff
@@ -83,4 +83,4 @@ GETTING HELP
 ------------
 
 For support, please see http://theforeman.org/support.html, the
-foreman-users@googlegroups.com mailing list or #theforeman on Freenode.
+foreman-users@googlegroups.com mailing list or #theforeman on Libera.Chat.
```

Comment:
the pointer to googlegroups should be updated as well, but that's outside the scope for this PR